core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/StrStatus.java

```diff
@@ -0,0 +1,48 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
```

Comment:
Should probably be either in core.data.types, or a static member of SampleStock since that's the only place it's used